autoload/airline/parts.vim

```diff
@@ -66,7 +66,7 @@ endfunction
 
 function! airline#parts#spell()
   let spelllang = g:airline_detect_spelllang ? printf(" [%s]", toupper(substitute(&spelllang, ',', '/', 'g'))) : ''
-  if g:airline_detect_spell && &spell
+  if g:airline_detect_spell && (&spell || (exists('g:airline_spell_check_command') && eval(g:airline_spell_check_command)))
```

Comment:
what is the `eval` for?

Reply:
To check custom spellcheck plugin status:
Set `g:airline_spell_check_command` as a string, and evaluate it every time.

For arbitrary spellcheck plugins, it should ask users to set their spellchecker's status check command.

Reply:
You mean it should be a function or something?
I just thought writing a function might be a bit bothering for users, rather than setting a command-string.